src/cgcomp.cpp

```diff
@@ -1159,29 +1148,26 @@ RC DOAS::oa_BegSubhr()
 {
 	RC rc = RCOK;
 	// Get total air flow rates and calculate mixed exhaust air state
-	oa_supAF.af_amf = 0.;
-	oa_exhAF.af_amf = 0.;
+	oa_supAF.af_Init();
+	oa_exhAF.af_Init();
 	float supVf = 0.f;
-	float exhVf = 0.f;
+	// float exhVf = 0.f;  not used
```

Comment:
This should just be removed instead of commented.

Reply:
Expanded comments and retained.